aimmo-game/simulation/world_map.py

```diff
@@ -213,9 +213,9 @@ def can_move_to(self, target_location):
             return False
         cell = self.get_cell(target_location)
 
-        return (cell.habitable
-                and (not cell.is_occupied or cell.avatar.is_moving)
-                and len(cell.moves) <= 1)
+        return (cell.habitable and
+                (not cell.is_occupied or cell.avatar.is_moving) and
+                len(cell.moves) <= 1)
```

Comment:
The former style is actually preferred by the newer versions of PEP8 (as they should be) - some tools just haven't updated to reflect this.